frontend/src/components/BCDataGrid/BCDataGridServer.jsx

```diff
@@ -219,33 +247,44 @@ BCDataGridServer.defaultProps = {
   gridRef: null,
   gridKey: `bcgrid-key-${Math.random()}`,
```

Comment:
Math.random() seems like it could result in some weird re-renders. Is there a way to make this more explicit?

Reply:
gridKey is provided as the required parameter, so it can be overrided with a unique key as requried.